lib/Cargo.toml

```diff
@@ -30,7 +30,7 @@ liboverdrop = "0.1.0"
 libsystemd = "0.7"
 openssl = "^0.10.64"
 # TODO drop this in favor of rustix
-nix = { version = "0.29", features = ["ioctl", "sched"] }
+nix = { version = "0.29", features = ["ioctl", "sched", "fs"] }
```

Comment:
I think this change may have only been needed in an earlier PR version? I dropped it...as the TODO says ideally we port to rustix.